config/serverless.oblt.yml

```diff
@@ -22,3 +22,8 @@ xpack.serverless.plugin.developer.projectSwitcher.currentType: 'observability'
 
 ## Disable adding the component template `.fleet_agent_id_verification-1` to every index template for each datastream for each integration
 xpack.fleet.agentIdVerificationEnabled: false
+
+## Required for force installation of APM Package
+xpack.fleet.packages:
+  - name: apm
+    version: latest
```

Comment:
For this to work in development we also need the `prerelease: true` flag, right?

Reply:
yes, that should be handled in `kibana.dev.yml` file